packages/app-project/stores/User/UserPersonalization/Notifications/helpers/getTalkUnreadConversationsIds.js

```diff
@@ -0,0 +1,31 @@
+import { talkAPI } from '@zooniverse/panoptes-js'
+
+export default async function getUnreadConversationsIds (authorization) {
+  if (!authorization) return undefined
+
+  let unreadConversationsIds = []
+
+  const getConversations = async (page = 1) => {
```

Comment:
Small code style comment: at the top of this file you declare a named function but here you an assign an anonymous function to a named variable. Is there a reason not to declare `async function getConversations(page = 1)` here?

I’m just curious to know why you’re using two different styles to declare functions in the same file.

Reply:
No good reason, picked this up and put it down a few months later and didn't catch, thank you for noting!